30413/Balacescu Ionut-Vlad/Sem2Assig1/main.c

```diff
@@ -0,0 +1,192 @@
+#include "head.h"
+#include <string.h>
+
+
+int main()
+{
+    initializeList();
+    /*AF(5);
+    AF(6);
+    AF(21);
+    AL(12);
+    DF();
+    DL();
+    DE(3);
+    DE(5);
+    AF(2);
+    AL(200);
+    PRINT_ALL();
+    AF(1);
+    PRINT_ALL();
+    PRINT_F(2);
+    PRINT_L(3);
+    DOOM_THE_LIST();
+    DL();
+    AF(42);
+    AL(24);
+    AF(9);
+    PRINT_ALL();
+    PRINT_L(2);*/
+
+    FILE *f=fopen("input.txt","rw");
+    FILE *o=fopen("output.txt","rw");
+    char *s1=(char*)malloc(sizeof(char)*100);
+    char *s2=(char*)malloc(sizeof(char)*100);
+    ///char aux[100];
+    while(fscanf(f,"%s",s1)!=EOF)
+    {
+        int nr=0;
+        if(strstr(s1,"AF")!=0)
+        {
+            for(int i=0; i<strlen(s1); i++)
+            {
+                if(s1[i]>='0'&&s1[i]<='9')
```

Comment:
I think you should have just used `strtok`, or at least extract this approach in a separate function as it repeats itself quite a few times